ctapipe/core/tool.py

```diff
@@ -229,6 +239,9 @@ def run(self, argv=None):
             with open("provenance.log", mode="w+") as provlog:
```

Comment:
This opens a hardcoded file and overwrites it without any warning. 
This is probably not really what we want here, right?

Reply:
probably not, it was just a hack for now to get the provenance info somewhere. 

 At minimum it should probably append, not overwrite, and perhaps the name should be a config option, or at least prepend the tool name (so you get `mytool-provenance.log`).   In the future, it will just be attached to output files, but so far that is not implemented.